src/styles/calendar.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  margin: 0 auto;
```

Comment:
You can't use margins for BEM blocks, please, use mix

Reply:
agree, let's fix it